### DISCUSSION TITLE
v1.6.4

### DISCUSSION THREAD

**Discussion by LazyImmortal** [LazyImmortal/Sesame#16] [Announcements]
## 更新说明
- 优化 验证逻辑(支持更多本地模式)
- 添加 只收能量时间段的日志输出
- 调整 TargetSDK到34
- 更新 Gradle版本
- 升级 编译JDK版本到21
- 添加 验证信息说明
- 修复 已知问题
- 替换 过时代码

## 下载说明

- Normal 为正常版本，适用于**Android 8.0**及以上的系统
- Compatible 为兼容版本，适用于**Android 7.0**及以下的系统，最低支持**Android 5.1**

<hr /><em>This discussion was created from the release <a href='https://github.com/LazyImmortal/Sesame/releases/tag/v1.6.4'>v1.6.4</a>.</em>

**Comment by sheepeeing** [comment]
还是用shizuku+LSPatch吗？修补后不能启动芝麻粒哇

**Comment by Abysslyz** [comment]
使用shizuku+LSPatch，本地模式提示不可用安装包，集成模式提示不可用修补包。

**Comment by 470596** [comment]
开发者是用哪个版本的支付宝

**Comment by tangjie602** [comment]
这个现在要怎么用阿，求告知，按照以前方式内置了打不开
